src/jsonrepair.ts

```diff
@@ -603,6 +609,17 @@ export function jsonrepair(text: string): string {
     }
   }
 
+  function expectDigitOrAppendToEnd(start: number, appendix = '0') {
```

Comment:
One small remark here: the optional parameter `appendix` makes the function more generic and suggests that there are different uses of it, but the parameter is not used in practice. Shall we move it into a `const` in the function (and maybe name it `suffix`)? 

And another naming suggestion: how about the name `expectDigitOrRepair`?

Reply:
Very valid points. Both is absolutely fine for me. At the time of writing the function I was not yet aware that it will actually replace all usages of expectDigit and that "0" works for all the cases I found. I started with the inlined `+ '0'` and then hit Ctrl-Alt-P :-)  

Do you want me to make those changes or do you want to do them yourself? Just let me know.

Reply:
Never mind, I just pushed the changes.

Reply:
👌 thanks, this is a nice improvement!